packages/collector/test/metrics/app/test.js

```diff
@@ -11,13 +11,19 @@ const _ = require('lodash');
 const expect = require('chai').expect;
 const { execSync } = require('child_process');
 const path = require('path');
+const supportedVersion = require('@instana/core').tracing.supportedVersion;
 
 const config = require('../../../../core/test/config');
 const { retry } = require('../../../../core/test/test_util');
 const ProcessControls = require('../../test_util/ProcessControls');
 const globalAgent = require('../../globalAgent');
 
-describe('snapshot data and metrics', function () {
+// Note: we can disable this test for prerelease versions beacause
+//       native modules won't be compiled successfully in prerelease node versions
```

Comment:
This test is not only testing native modules.
Its better to devide this test into two parts:

One it -> general checks
Second it -> native module checks